pkg/util/etcdutil/start_etcd.go

```diff
@@ -46,7 +46,7 @@ spec:
   containers:
   - command:
     - /bin/sh
-    - -c
+    - -ec
```

Comment:
Yeah, if we don't need separate shell - seems simpler to just execute the binary directly.